drivers/spi/spi_context.h

```diff
@@ -245,6 +245,25 @@ bool spi_context_rx_on(struct spi_context *ctx)
 	return !!(ctx->rx_buf || ctx->rx_len);
 }
 
+static ALWAYS_INLINE
+int spi_context_get_longest_single_xfer_buffers(struct spi_context *ctx,
+						void **tx_buf,
+						void **rx_buf)
+{
+	*tx_buf = ctx->tx_buf;
```

Comment:
I thought that it would give me better encapsulation. I do not reference to spi_context data directly, I just call functions that operate on spi_context from SPI driver code.